core/src/main/scala/com/evolutiongaming/kafka/flow/RebalanceListener.scala

```diff
@@ -0,0 +1,62 @@
+package com.evolutiongaming.kafka.flow
+
+import cats.data.NonEmptySet
+import cats.syntax.all._
+import com.evolutiongaming.catshelper.{BracketThrowable, LogOf}
+import com.evolutiongaming.kafka.flow.ConsumerFlow.log
+import com.evolutiongaming.kafka.flow.kafka.Consumer
+import com.evolutiongaming.skafka.{Partition, Topic, TopicPartition}
+import com.evolutiongaming.skafka.consumer.{RebalanceListener => SRebalanceListener}
+
+object RebalanceListener {
+
+  def apply[F[_]: BracketThrowable: LogOf](
```

Comment:
Here you can replace `BracketThrowable` with `Monad` now and write `State` based unit test.

Reply:
Done